### PR TITLE
ref bug#134533, bug#133159 Fix reload data when visible hint

### PR DESCRIPTION
**Pull request by tranminhtri911**
- ref bug#134533, bug#133159 Fix reload data when visible hint
- https://dev.framgia.com/redmine/issues/134553
- https://dev.framgia.com/redmine/issues/133159
- Reload lại dữ liệu khi đang ở màn hình đó
- Effect : List request personal, request manager

### CODE REVIEW

File: app/src/main/java/com/framgia/wsm/screen/managelistrequests/ManageListRequestsViewModel.java [hunk 1]
```diff
@@ -159,25 +161,62 @@ public void onGetListRequestManageSuccess(Object object, boolean isLoadMore) {
     }
 
     @Override
-    public void onReloadData() {
+    public void setCurrentStatusFromNotifications(int permissionType, String trackableStatus) {
+        if (permissionType == 0) {
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/screen/managelistrequests/ManageListRequestsViewModel.java]: ERROR: [PMD-CPD] 25 lines duplicated.
app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestViewModel.java

Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/screen/managelistrequests/ManageListRequestsViewModel.java]: pls declare permissionType is a JavaAnnotation like ``StatusCode``

File: app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestViewModel.java [hunk 2]
```diff
@@ -207,7 +210,7 @@ public void onGetListRequestSuccess(int requestType, Object object, boolean isLo
 
     @Override
     public void setCurrentStatusFromNotifications(int permissionType, String trackableStatus) {
-        if (permissionType == 1) {
+        if (permissionType == NotificationViewModel.PermissionType.MANAGER) {
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestViewModel.java]: ERROR: [PMD-CPD] 25 lines duplicated.
app/src/main/java/com/framgia/wsm/screen/managelistrequests/ManageListRequestsViewModel.java

File: app/src/main/java/com/framgia/wsm/screen/managelistrequests/ManageListRequestsViewModel.java [hunk 3]
```diff
@@ -159,25 +162,62 @@ public void onGetListRequestManageSuccess(Object object, boolean isLoadMore) {
     }
 
     @Override
-    public void onReloadData() {
+    public void setCurrentStatusFromNotifications(int permissionType, String trackableStatus) {
+        if (permissionType == NotificationViewModel.PermissionType.USER) {
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/screen/managelistrequests/ManageListRequestsViewModel.java]: ERROR: [PMD-CPD] 25 lines duplicated.
app/src/main/java/com/framgia/wsm/screen/listrequest/ListRequestViewModel.java

**Pull request merged**